siloLapisTests/test/aggregated.spec.ts

```diff
@@ -104,6 +104,18 @@ describe('The /aggregated endpoint', () => {
     expect(descendingOrderedResult.data[0]).to.have.property('division', 'Zürich');
   });
 
+  it('should return bad request for non existing fields on GET request', async () => {
+    const urlParams = new URLSearchParams({
+      fields: 'notAField',
+    });
+
+    const result = await getAggregated(urlParams);
+
+    expect(result.status).equals(400);
+    const resultJson = await result.json();
+    expect(resultJson.error.detail).to.include('Unknown field: notAField, known values are [primaryKey,');
```

Comment:
Actually I think we should also be able to see this in a unit test now? Should we add one?